setup.cfg

```diff
@@ -0,0 +1,7 @@
+[pytest]
```

Comment:
:+1: It would be ideal to integrate linting with pep8 into our CI pipeline. We might also consider adding a git hook.